schema/synthetix/materialized_view_trades.sql

```diff
@@ -0,0 +1,268 @@
+BEGIN;
```

Comment:
This is definitely wrong, but I created it as a starting point if we need to use a materialized view here